praktikumsplaner-frontend/frontend/src/stores/error.ts

```diff
@@ -0,0 +1,30 @@
+import { defineStore } from "pinia";
+
+export interface ErrorState {
```

Comment:
with an explicit error store, why don't you remove the level `ERROR`?

Reply:
The store is only meant for errors produced directly by user inputs. It is now renamed to avoid confusion.